lib/http.ex

```diff
@@ -117,7 +117,13 @@ defmodule Braintree.HTTP do
 
   defp base_options do
     path = Path.join(:code.priv_dir(:braintree), @cacertfile)
-
-    [hackney: [ssl_options: [cacertfile: path]]]
+    timeout = Braintree.get_env(:timeout, 8000) # HTTPoison default
```

Comment:
I'm not set on it, but it would be nice to pull these values out into module attributes that identify them.